src/DateTime.php

```diff
@@ -276,9 +276,8 @@ public function diff($datetime, $absolute = false)
      * Returns date formatted according to given format.
      *
      * @param string $format
-     * @return string
      */
-    public function format($format)
+    public function format($format): string
```

Comment:
You may need to add a type hint to string here as well to match the signature from https://www.php.net/manual/en/datetime.format.php